windowsdoc.md

```diff
@@ -127,6 +128,15 @@ provide a custom timeout period using `--eval-timeout` argument. So if you face
 this error, the best option is to install Nodejs on your local system. If you
 can't then use the `--eval-timeout` argument and set a higher timeout value.
 
+### Fatal error occurs when executing a python command Line instruction as a job on python version 3.
+
+If your workflow creates a python command line instruction (like we do in this
+unit test ) and you get a error `Fatal Python error: Failed to initialize Windows
+ random API (CryptoGen)` then try setting `SYSTEMROOT` environment variable to
+windows directory location (usually at `C:\\WINDOWS`). The reason behind this error
+is that when a python instance is created from current instance, it needs `SYSTEMROOT`
+env variable to starts its random generator on Windows OS.
```

Comment:
Is this a bug with Python, Virtualenv, or us? Do you have any external references for this?